examples/docs_snippets/docs_snippets/concepts/assets/graph_backed_asset.py

```diff
@@ -88,7 +88,7 @@ def downstream_asset(middle_asset):
 
 basic_deps_job = define_asset_job(
     "basic_deps_job",
-    AssetSelection.assets(upstream_asset, middle_asset, downstream_asset),
+    AssetSelection.assets(upstream_asset, middle_asset, downstream_asset),  # type: ignore
```

Comment:
why is this a type error?

Reply:
See previous comment, the type of `middle_asset` is not well understood by the type checker. FWIW I believe we should not recommend this pattern for this reason but the docs prose specifically suggests this so I did not change the variable name.

Reply:
got it - agree and makes sense